doc/source/skips/1-governance.rst

```diff
@@ -145,16 +146,18 @@ are made according to the following rules:
   sentence (but no change of the scikit-image.org landing page or the “about”
   page), require approval by a core developer *and* no disagreement or requested
   changes by a core developer on the issue or pull request page (lazy
-  consensus). Core developers are expected to give “reasonable time” to others
-  to give their opinion on the pull request if they’re not confident others
-  would agree.
+  consensus). Core developers are expected to wait one or two days for others to give
+  their opinion if they’re not confident that others would agree.
```

Comment:
Do the two instances of "others" above mean "other core developers?"

Reply:
Yes. Though I imagine that in practice and most cases any concerns by non-core devs would also be treated with the same politeness.

Reply:
Right. Then let's not have it mean "other core developers." Thanks. The sentence is not the most beautiful but I'll settle. Maybe "voice" instead of "give?"

Reply:
> Then let's not have it mean "other core developers."

It's not clear to me, should I update it or are you happy with this?

Reply:
I'm happy as is. If we meant "other core developers," I would have suggested:
```diff
-others
+other core developers
```
(to be explicit/unambiguous).